src/app/profile/shared/profile.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Lot } from '@app/shared/models/lot.model';
+import { environment } from '@env/environment';
+
+import { ProfileModule } from '../profile.module';
+
+@Injectable({ providedIn: ProfileModule })
+export class ProfileService {
+  private backUrl = `${environment.apiUrl}/api/lots`;
```

Comment:
`lotsUrl` is slightly more appropriate name for this constant

Reply:
done